js/src/components/paid-ads/validateCampaign.test.js

```diff
@@ -61,15 +68,84 @@ describe( 'validateCampaign', () => {
 		let errors;
 
 		values.amount = 0;
-		errors = validateCampaign( values );
+		errors = validateCampaign( values, validateCampaignOptions );
 
 		expect( errors ).toHaveProperty( 'amount' );
 		expect( errors.amount ).toMatchSnapshot();
 
 		values.amount = -0.01;
-		errors = validateCampaign( values );
+		errors = validateCampaign( values, validateCampaignOptions );
 
 		expect( errors ).toHaveProperty( 'amount' );
 		expect( errors.amount ).toMatchSnapshot();
 	} );
+
+	it( 'When a budget is provided and the amount is less than the minimum, should not pass', () => {
+		const mockFormatAmount = jest.fn().mockReturnValue( 'Rs 30' );
+		values.amount = 10;
+
+		const opts = {
+			dailyBudget: 100,
+			formatAmount: mockFormatAmount,
+		};
+
+		const errors = validateCampaign( values, opts );
+
+		expect( errors ).toHaveProperty( 'amount' );
+		expect( errors.amount ).toContain( 'is at least Rs 30' );
+	} );
+
+	it( 'When a budget is provided and the amount is slightly less than the minimum, should not pass', () => {
+		const mockFormatAmount = jest.fn().mockReturnValue( 'Rs 30' );
+		values.amount = 29.99;
+
+		const opts = {
+			dailyBudget: 100,
+			formatAmount: mockFormatAmount,
+		};
+
+		const errors = validateCampaign( values, opts );
+
+		expect( errors ).toHaveProperty( 'amount' );
+		expect( errors.amount ).toContain( 'is at least Rs 30' );
+	} );
+
+	it( 'When a budget is provided and the amount is same as the minimum, should pass', () => {
+		const mockFormatAmount = jest.fn().mockReturnValue( 'Rs 30' );
+		values.amount = 30;
+
+		const opts = {
+			dailyBudget: 100,
+			formatAmount: mockFormatAmount,
+		};
+
+		const errors = validateCampaign( values, opts );
+		expect( errors ).not.toHaveProperty( 'amount' );
+	} );
+
+	it( 'When a budget is provided and the amount is greater than the minimum, should pass', () => {
+		const mockFormatAmount = jest.fn().mockReturnValue( 'Rs 35' );
+		values.amount = 35;
+
+		const opts = {
+			dailyBudget: 100,
+			formatAmount: mockFormatAmount,
+		};
+
+		const errors = validateCampaign( values, opts );
+		expect( errors ).not.toHaveProperty( 'amount' );
+	} );
+
+	it( 'When a budget is provided and the amount is slightly greater than the minimum, should pass', () => {
+		const mockFormatAmount = jest.fn().mockReturnValue( 'Rs 35' );
```

Comment:
Although this return string doesn't affect the test result, 35 is not the actual minimum value.

Reply:
Updated the mock to return the actual minimum value.